imu_sensor_node:
  ros__parameters:
    device_path: "/dev/ttyUSB0" # Path to receiver port
    publish_interval: 500 # publish interval in milliseconds
    baud_rate: 115200 # baud rate